src/main/webapp/static/js/controller/gameController.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+angular.module('myApp').controller('GameController', ['$scope', '$log' ,'GameService', function($scope, $log, GameService) {
```

Comment:
So $scope is never recommended because you don't want certain variables and functions to live in a global scope. I would check out the Johnpapa style guide and rewrite some of this controller using the information there